seed/static/seed/js/services/inventory_service.js

```diff
@@ -54,14 +54,19 @@ angular.module('BE.seed.service.inventory', []).factory('inventory_service', [
 
       var params = {
         organization_id: organization_id,
-        page: page,
-        per_page: per_page || 999999999,
         include_related: include_related,
         ids_only: ids_only,
         ...format_column_sorts(column_sorts),
         ...format_column_filters(column_filters),
       };
 
+      if (ids_only) {
```

Comment:
The endpoint wont allow ids_only and page/per_page at the same time.

Reply:
Nice!